src/parser/jobs/dnc/modules/Gauge.tsx

```diff
@@ -135,12 +140,38 @@ export class Gauge extends CoreGauge {
 		this.addEventHook(statusApplyFilter.status(espritStatusMatcher), this.addEspritGenerationHook)
 		this.addEventHook(statusRemoveFilter.status(espritStatusMatcher), this.removeEspritGenerationHook)
```

Comment:
Can we also unhook these once we know we have gauge events, and clear the esprit partner damage hook if it exists?  Should also mean that we wouldn't need to do checks in the probabilistic esprit generation (onDamage) to bail if this log has gauge events.

Reply:
The remaining generation event stuff is used to calculate parts of the overcap suggestion for when the log events show the gauge is capped but we're getting another event that should have generated Esprit or Feathers. Gauge changed events can't get that information for us, so we still need the generation events to infer it.